x/adminmodule/abci.go

```diff
@@ -68,4 +69,62 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 		)
 		return false
 	})
+
+	keeper.IterateActiveProposalsQueue(ctx, func(proposal v1.Proposal) bool {
+		var tagValue string
```

Comment:
unused var

Reply:
this is not true (mb IDE is lying to you?), due this var is used to emit event at the end

Reply:
hm, but it's not set anywhere like above:
`tagValue = govtypes.AttributeValueProposalPassed`